shared/domain/src/main/java/com/ivy/domain/features/IvyFeatures.kt

```diff
@@ -32,6 +32,13 @@ class IvyFeatures @Inject constructor() : Features {
         defaultValue = true
     )
 
+    override val showCategorySearchBar = BoolFeature(
+        key = "search_categories",
+        name = "Search categories",
+        description = "Show search bar in category screen",
+        defaultValue = false
```

Comment:
Let's enable this feature by default 🙌

Reply:
@shamim-emon enable it bt default and we can merge 🚀 Ping me when it's ready (CI passed)

Reply:
@ILIYANGERMANOV It's  ready!